lib/gui/modules/drive-scanner.js

```diff
@@ -110,9 +97,22 @@ const scan = () => {
       includeSystemDrives: settings.get('unsafeMode')
     },
     usbboot: {
-      files: {
-        'bootcode.bin': USBBOOT_BOOTCODE_BIN_BUFFER,
-        'start.elf': USBBOOT_START_ELF_BUFFER
+      readFile: (name) => {
+        const blobPath = _.attempt(() => {
```

Comment:
This blobPath stuff can be moved to after the `if (BLOBS_CACHE[name]) {` block :)

Reply:
True. Fixed!